satpy/readers/tropomi_l2.py

```diff
@@ -206,9 +206,25 @@ def get_dataset(self, ds_id, ds_info):
         file_key = ds_info.get('file_key', ds_id.name)
         data = self[file_key]
         data.attrs = self.get_metadata(data, ds_info)
-        fill = data.attrs.pop('_FillValue')
+        fill_value = data._FillValue
```

Comment:
I know I did this in the other reader, but do we maybe not want to assume that `_FillValue` will exist? Maybe changing the `pop` to `get` and default to `np.nan`?

Reply:
@djhoese If we use `pop`, the `_FillValue` won't exist in the data anymore. Users don't know how to deal with it, right?

Reply:
I meant change it to `fill_value = data.attrs.get('_FillValue', np.nan)` or something similar.

Reply:
Or if the `_FillValue` needs to be removed, `fill_value = data.attrs.pop('_FillValue', np.nan)`.